sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_responses.hpp

```diff
@@ -31,12 +30,29 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake { nam
     Storage::Metadata Metadata;
   };
 
-  using CreateFileSystemResult = FileSystemCreateResult;
+  struct CreateFileSystemResult
+  {
+    bool Created = true;
+    std::string ETag;
+    Core::DateTime LastModified;
+    std::string NamespaceEnabled;
```

Comment:
what's the possible value of this field?

Reply:
True/False I suppose, but I haven't tested it yet.

Reply:
Why isn't it a `bool`?

Reply:
Actually, I'm going to remove this, since blob is not supporting this return value.

Reply:
Sure, removing it is fine.

My only concern is that you have several TODO items that don't have linked issues.

Reply:
I'll do it in this PR since it is raised as a comment in this PR. I think it justifies 'related to this PR'.

Reply:
Nah, I don't think it justifies the relevance. I'm going to raise more comments. Like some const variables still have `c_` prefix.

Reply:
Then I think it's best to create issues to track those.

Reply:
Then create an issue